WcaOnRails/lib/custom_i18n_scanner.rb

```diff
@@ -42,12 +52,9 @@ def scan_file(path)
       if !input_params.include?("label:")
         # Simple form can fetch its labels from activerecord.attributes,
         # Mark it as used ... Except if the model is not an ActiveRecord ;)
-        # if not we mark the label as used!
-        retval << if active_record?(model)
-                    ["activerecord.attributes.#{model}.#{attribute}", occurrence]
-                  else
-                    ["simple_form.labels.#{model}.#{attribute}", occurrence]
-                  end
+        if active_record?(model)
+          retval <<  ["activerecord.attributes.#{model}.#{attribute}", occurrence]
+        end
```

Comment:
@viroulep The only non-activerecord models are the contact forms at this point. Those have their attributes listed [here](https://github.com/jonatanklosko/worldcubeassociation.org/blob/4c98fe7e8ddca363061a8cb0a3632efb0be37e31/WcaOnRails/config/locales/en.yml#L370-L379) (`simple_for.labels.*` are not used) and this node is ignored [here](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/config/i18n-tasks.yml.erb#L128). So I think that it makes sense to do something only if the model is activerecord.